geometry/create-and-edit-geometries/README.metadata.json

```
@@ -0,0 +1,37 @@
+{
+  "category": "Geometry",
+  "description": "Use the Geometry Editor to create new point, multipoint, polyline, or polygon geometries or to edit existing geometries by interacting with a map view.",
+  "ignore": false,
+  "images": [
+    "CreateAndEditGeometries.png"
+  ],
+  "keywords": [
+    "draw",
+    "edit",
+    "freehand",
+    "geometry editor",
+    "sketch",
+    "vertex",
+    "Geometry",
+    "GeometryEditor",
+    "Graphic",
+    "GraphicsOverlay",
+    "MapView"
+  ],
+  "redirect_from": [
+    "/java/latest/sample-code/create-and-edit-geometries.htm"
+  ],
```

```suggestion
  "redirect_from": "",
```
No need for a redirect